src/main/java/com/kaleido/fetch/service/ActivityService.java

```diff
@@ -177,5 +187,34 @@ private Activity buildActivity(Experiment experiment) {
                 .description(experiment.getDescription())
                 .build();
     }
+   
+    private List<PlateMap> getPlateMapSummaryFromCabinet(String activityName) {
+        RestTemplate restTemplate = new RestTemplate();
+   	    String cabinetplateInfoURI = cabinetURI + "plate-map-summary/"+activityName;
+   	    ResponseEntity<List<PlateMap>> plateMapResponse =restTemplate.exchange(cabinetplateInfoURI,
```

Comment:
This is fine for now but we will probably want to create another ticket to refactor this to use the cabinet client

Reply:
create a ticket , we will work on Cabinet client

Reply:
I updated direct call to cabinet with cabinet client